dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/redshift/RedshiftClusterNodesTask.java

```diff
@@ -58,4 +58,16 @@ protected Void doRun(TaskRunContext context, @Nonnull ByteSink sink, Handle hand
             .collect(toList()));
     return null;
   }
+
+  private String getRequestDescription() {
+    return "AmazonRedshift.describeClusters";
+  }
+
+  @Override
+  public String toString() {
```

Comment:
Please adapt this class to the new `describeSourceData` method (after my recent merge of #344).

Reply:
👍